### ISSUE TITLE
API Readiness errors and questions for management plane

### ISSUE THREAD

**Issue by ladonnaq** [Azure/azure-sdk-tools#5964]
I am working with the Palo Alto service team to use the release planner app to validate public preview API readiness for their initial beta SDK releases.  I have encountered some errors, questions related to function, etc.  

- [ ] API readiness validation blocked by breaking changes.  PR used was https://github.com/Azure/azure-rest-api-specs/pull/23435.  
This scenario is confusing because there are 2 PRs in the private repo which have approved breaking changes for changes that are in this PR for the public repo.  See comment from service team below. 
_"Please provide waiver for Breaking Changes. We are adding changes to the existing api-version as our service is still in private preview. Hence, new api version is not required. These changes are already approved in private repo as well -
https://github.com/Azure/azure-rest-api-specs-pr/pull/11696
https://github.com/Azure/azure-rest-api-specs-pr/pull/11240"_ 

![image](https://user-images.githubusercontent.com/67656762/231548143-6974f3af-622f-420a-8473-0cdc81edf31c.png)

- [ ] Can't input text into review ticket field to add link to the GitHub issue for namespace review by arch board for initial beta => https://github.com/Azure/azure-sdk-tools/issues/6053
![image](https://user-images.githubusercontent.com/67656762/231553812-541f4798-65f0-4e30-8a12-5200f9f76639.png)

- [ ] The SDKs should NOT be generated in the API Readiness application for management plane or data plane.  I believe there is a GitHub issue open for this but just wanted to make sure that we move this asap. The service teams I have been working with found this confusing.  SDK Breaking changes are reviewed as part of API readiness for management plane BUT since the information relating to todo's for SDK release are in the SDK release app we should not generate and display the SDKs in the API Readiness app.  

- [ ] @josefree @maririos Question about what is required for a API Spec PR merge and how to provide guidance on which PR a service team should use for API readiness.  I noticed that PR 23435 is merged but the ARM sign off label is not on the PR so an error is displayed in the API Readiness app that ARM has not signed off.   I noticed that ARM signed off the 2 PRs that were in the private repo and are part of this version of the API Spec.  So which PR should be used to validate API Readiness?  For public preview today we ask teams to send us the link to the either the PR in the private repo that is related to the API Spec that will be published for public preview (for services that are new and do not want to have anything in the public repo until announce). For other scenarios, we ask for the last PR that is associated with the API Spec version for the public launch for the product.   

- [x] #5976

This the PR that was used to publish the API spec from the private repo to the public repo in preparation to release the SDKs for public preview.   https://github.com/Azure/azure-rest-api-specs/pull/22878

This is the PR I would expect they would use for API Readiness.  However, there is also another PR so I tried that one as well. It also gave me the same error - https://github.com/Azure/azure-rest-api-specs/pull/23435.  

![image](https://user-images.githubusercontent.com/67656762/231542397-49b0e694-8df1-4619-b5d8-4b9a575592b6.png)
